nixos/modules/hardware/network/intel-2200bg.nix

```diff
@@ -23,7 +23,7 @@
 
   config = lib.mkIf config.networking.enableIntel2200BGFirmware {
 
-    hardware.enableAllFirmware = true;
+    hardware.firmware = [ pkgs.intel2200BGFirmware ];
```

Comment:
I'm not 100% sure on the behavior if `hardware.enableAllFirmware` is also enabled. The firmware should probably be appended to the list instead.